example/views.py

```diff
@@ -200,7 +208,18 @@ def get_queryset(self, *args, **kwargs):
         return super(CommentViewSet, self).get_queryset()
 
 
-class CompanyViewset(PreloadIncludesMixin, viewsets.ModelViewSet):
+class TestModelViewSet(PreloadIncludesMixin, RelatedMixin, viewsets.ModelViewSet):
```

Comment:
Changing to DRF `ModelViewSet` shouldn't have happened (somehow I have overlooked this.)

So simply derive `CompanyViewset` from DJA `ModelViewSet` should do as it was before. Does it work then?

Reply:
@sliverc No, that's the problem I've referred to @Anton-Shutik above: https://github.com/django-json-api/django-rest-framework-json-api/pull/643#issuecomment-497753149 because DJA's ModelViewSet pulls in two (now) conflicting Mixins.

Reply:
https://github.com/django-json-api/django-rest-framework-json-api/pull/651 should address this issue. Once it is merged best to rebase this PR and remove the workaround.

Reply:
After a few git mistakes I've removed the workaround and rebased.